dae-alright/src/components/modal/custom-modal/custom-modal.component.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from "react";
+import Modal from "react-bootstrap/Modal";
+import { Button } from "react-bootstrap";
+import "./custom-modal.styles.scss";
+
+const CustomModal = props => {
+  const [show, setShow] = useState(false);
+
+  const handleClose = () => setShow(false);
+  const handleShow = () => setShow(true);
+
+  return (
+    <>
+      <Button variant="primary" onClick={handleShow}>
+        Sign-Up now
+      </Button>
+
+      <Modal show={show} onHide={handleClose} className="custom-modal">
```

Comment:
@John-pels  this `show` should be a props and you can remove `handleClose` and `handleShow`

Reply:
I have removed it.